app/models/manageiq/providers/dummy_provider/inventory/persister/target_collection.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::DummyProvider::Inventory::Persister::TargetCollection < ManageIQ::Providers::DummyProvider::Inventory::Persister
+  def initialize_inventory_collections
+    add_collection(cloud, :flavors)
+    add_collection(cloud, :vms)
+  end
+
+  def targeted?
+    true
+  end
+
+  def parent
+    manager.presence
+  end
+
+  def shared_options
```

Comment:
shared options are now defined in core, so it can be skipped.
But on the other side, I would rather call `super` there, because it's dummy provider and new developer should know about this option. 
(I think I had to rename `shared_options` to `shared_ic_properties` or something like that during refactoring, `shared_options` seems to be little confusing - so add short comment to this method, please)